server/src/inmantals/server.py

```diff
@@ -627,7 +626,7 @@ async def textDocument_hover(self, textDocument, position):
 
         data = list(range)[0].data
         docstring = textwrap.dedent(data.docstring.strip("\n")) if data.docstring else ""
-        docstring = docstring.replace(" ", "&nbsp;")
+        docstring = docstring.replace(" ", "&nbsp;").replace("\n", "\n\n").strip()
```

Comment:
I noticed that the formatting was not always correct:
before:
![Screenshot from 2023-04-28 14-06-51](https://user-images.githubusercontent.com/95339363/235184872-bd1b8e3b-f809-4172-8e8e-907744f9ce87.png)

after:
![Screenshot from 2023-04-28 14-08-49](https://user-images.githubusercontent.com/95339363/235184901-3940d1b0-258c-4222-9030-f1ddd88a01bf.png)